src/SFA.DAS.AssessorService.Application/Handlers/Certificates/GetCertificatesHistoryHandler.cs

```diff
@@ -71,7 +71,7 @@ private async Task<PaginatedList<CertificateSummaryResponse>> MapCertificates(Pa
                         recordedBy = certificate.CertificateLogs
                             .OrderByDescending(q => q.EventTime)
                             .FirstOrDefault(certificateLog =>
-                                certificateLog.Status == Domain.Consts.CertificateStatus.Submitted)?.Username;
+                                certificateLog.Action == Domain.Consts.CertificateActions.Submit)?.Username;
```

Comment:
no existing unit tests around this area?

Reply:
I have added a unit test for this scenario